package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "fluid-covid-map-viz",
+    "version": "0.0.2",
+    "description": "Sketch visualisation of covid data as map and table",
+    "main": "index.js",
+    "nyc": {
+        "exclude": [
+            "node_modules/**",
+            "tests/**"
+        ]
+    },
+    "scripts": {
```

Comment:
Shall we add a npm script "lint": `"lint": "grunt lint"`? When the underlying linting is moved away from `grunt`, the linting command will remain the same.

`How to lint` should be described in `README.md`.